GW2EIEvtcParser/EIData/ProfHelpers/Mesmer/VirtuosoHelper.cs

```diff
@@ -29,7 +29,9 @@ internal static class VirtuosoHelper
                 }
                 return true;
             }).WithBuilds(GW2Builds.October2022Balance),
-            new EffectCastFinder(BladeturnRequiem, EffectGUIDs.VirtuosoBladeturnRequiem).UsingSrcSpecChecker(Spec.Virtuoso).WithBuilds(GW2Builds.June2023Balance)
+            new EffectCastFinder(BladeturnRequiem, EffectGUIDs.VirtuosoBladeturnRequiem).UsingSrcSpecChecker(Spec.Virtuoso).WithBuilds(GW2Builds.June2023Balance),
+            new DamageCastFinder(ThousandCuts, ThousandCuts).UsingICD(5000).UsingNotAccurate(true).UsingDisableWithEffectData(),
```

Comment:
This should be removed, it is not possible to accurately time the cast time of the skill by damage only, could be wrong up to 5 seconds.